docs/components/organization/organization-list.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: <OrganizationList />
+description: The `<OrganizationList />` component is used to display organization related memberships, invitations, and suggestions for the user.
+---
+
+# `<OrganizationList />`
+
+<Images
+  alt="Organization List Example"
+  width={992}
+  height={568}
+  src="/docs/images/ui-components/component-org_list.png"
+/>
+
+The `<OrganizationList />` component is used to display organization related memberships, invitations, and suggestions for the user.
+
+## Usage
+
+<Tabs items={['Next.js', 'React', 'Remix']}>
+  <Tab>
+    <CodeBlockTabs options={["App Router", "Pages Router"]}>
```

Comment:
Let's use CodeBlockTabs instead of Tabs, and we can have the options be {["Next.js App Router", "Next.js Pages Router", "React", "Remix"]}

Tabs should be more for complex content.